state-chain/custom-rpc/src/lib.rs

```diff
@@ -260,7 +260,7 @@ pub struct IngressEgressEnvironment {
 	pub ingress_fees: HashMap<ForeignChain, HashMap<Asset, NumberOrHex>>,
 	pub egress_fees: HashMap<ForeignChain, HashMap<Asset, NumberOrHex>>,
 	pub witness_safety_margins: HashMap<ForeignChain, Option<u64>>,
-	pub minimum_egress_amounts: HashMap<ForeignChain, HashMap<Asset, NumberOrHex>>,
+	pub egress_dust_limits: HashMap<ForeignChain, HashMap<Asset, NumberOrHex>>,
```

Comment:
my thinking with minimum_egress_amounts was that it's more general and therefore future proof - while still quite clear what it is. not a strong opinion though